control/utils.py

```diff
@@ -182,6 +183,24 @@ def is_valid_nqn(nqn):
 
         return (0, os.strerror(0))
 
+class GatewayUtilsCrypto:
+    ENCRYPTION_KEY = b"Q\xba\x7f._i\x9f%\xed\x96\xe0\xaf\xab k\x9e\xd6\xb6c\xb8\xee\xf6\xa7\xb1s\xf7\xf4\x89f'\x85b"
+    secret_box = None
+
+    @staticmethod
+    def encrypt_text(msg : str) -> bytes:
+        if GatewayUtilsCrypto.secret_box == None:
+            GatewayUtilsCrypto.secret_box = nacl.secret.SecretBox(GatewayUtilsCrypto.ENCRYPTION_KEY)
+        encrypted = GatewayUtilsCrypto.secret_box.encrypt(msg.encode("utf-8"))
+        return encrypted
```

Comment:
This is a `@classmethod` (it doesn't use instance data, as regular instance method, but class variables) so it should be rewritten as this in order to comply with the encapsulation principle:
```suggestion
    @classmethod
    def encrypt_text(cls, msg : str) -> bytes:
        if cls.secret_box == None:
            cls.secret_box = nacl.secret.SecretBox(cls.ENCRYPTION_KEY)
        encrypted = cls.secret_box.encrypt(msg.encode("utf-8"))
        return encrypted
```

Reply:
OK